idb/postgres/internal/schema/setup_postgres.sql

```diff
@@ -117,3 +117,13 @@ CREATE TABLE IF NOT EXISTS account_app (
 
 -- For looking up existing app local states by account
 CREATE INDEX IF NOT EXISTS account_app_by_addr_partial ON account_app(addr) WHERE NOT deleted;
+
+-- For looking up app box storage
+CREATE TABLE IF NOT EXISTS app_box (
+  app bigint NOT NULL,
+  name bytea NOT NULL,
+  size smallint NOT NULL,     -- _NEGATIVE_ of box'es size
```

Comment:
I'm considering removing this column. If we never care about deleted box sizes, then there is no need for it. If we do, there are alternatives. Here's a copy paste of a recent [slack thread](https://algorand-internal.slack.com/archives/C020LLY8VBM/p1656447698751459?thread_ts=1656435858.591749&cid=C020LLY8VBM):

```
[John Jannotti]
size does not seem needed though, as I don’t expect queries based on size, and the size is derivable from the value itself, which is always full length.

[Zeph Grunschlag]
Regarding size
Suppose we want to be able to answer the following:
does app X have any deleted boxes?
If so, what are their names, and what were their sizes?
and in a sequence of create-replace-delete-create-replace-delete.. using the same name, we only care about the last existing box
If we use the approach that NULL value indicates deleted, then we can no longer query the length of the value for size.
An alternate approach (more space efficient unless there are MANY deleted boxes), is to add another bool column deleted and not set value <-- NULL when deleting.
If we don't care at all to know which boxes have been deleted, then the schema gets simplified and no size column is necessary
Or if we want to know about which boxes were deleted, but don't care about their size while existing, we can use the approach of NULL'ing the value with no size column.
```

Reply:
@shiqizng - I'm re-opening the resolved conversation with @jannotti . I found the argument to rename the original column `box_name` to `name` compelling since the table is `box_app` and so `box_app.name` seems pretty clear. However, Lamprey ought to set the naming conventions IMO. Let me know.